commit e3612e8dcb1a43bc358248845aa4f16f4a2792c7
Author: Stephan Hartmann <stha09@googlemail.com>
Date:   Fri Jun 30 19:15:09 2023 +0200

    remoting: remove unused dependency on libevdev

    There is only one include, but nothing of libevdev is used. Instead
    linux/input.h (defines BTN_*).

--- a/remoting/host/BUILD.gn
+++ b/remoting/host/BUILD.gn
@@ -241,10 +241,6 @@
   pkg_config("xkbcommon") {
     packages = [ "xkbcommon" ]
   }
-
-  pkg_config("libevdev") {
-    packages = [ "libevdev" ]
-  }
 }
 
 # This must be a static library instead of a source set because
@@ -576,7 +572,6 @@
       "x11_display_util.cc",
       "x11_display_util.h",
     ]
-    configs += [ ":libevdev" ]
     libs += [ "//third_party/libei/lib64/libei.a" ]
     public_deps += [
       "//third_party/wayland:wayland_client",
--- a/remoting/host/linux/remote_desktop_portal_injector.cc
+++ b/remoting/host/linux/remote_desktop_portal_injector.cc
@@ -7,9 +7,9 @@
 #include <gio/gio.h>
 #include <gio/gunixfdlist.h>
 #include <glib-object.h>
+#include <linux/input.h>
 #include <poll.h>
 
-#include <libevdev/libevdev-uinput.h>
 #include <utility>
 
 #include "base/check.h"
